lib/elixir/lib/code.ex

```diff
@@ -650,6 +650,10 @@ defmodule Code do
       when non-existing atoms are found by the tokenizer.
       Defaults to `false`.
 
+    * `:warn_on_unnecessary_quotes` - when `false`, does not warn
+      when atoms, keywords and calls have unnecessary quotes on
+      them
```

Comment:
I would add a dot at the end and the default value to keep consistency with previous descriptions.

> * `:warn_on_unnecessary_quotes` - when `false`, does not warn when atoms, keywords and calls have unnecessary quotes on them. Defaults to `true`.

Reply:
And maybe use and "or" instead of "and"? 

> does not warn when atoms, keywords **or** calls have unnecessary quotes on them